### ISSUE TITLE
Moving tiles around leads to total disaster

### ISSUE THREAD

**Issue by artworksmetal** [darktrojan/newtabtools#374]
I'm trying to create an elegant 6 x 8 grid of custom tiles and discovering that the operation can be quite buggy. I highly suggest backing up often. Moving tiles can cause tiles to completely disappear (along with the thumbnail). Or, tiles will just stop relocating and you have to re-load the page, restart Firefox, or go back to your last backup.
Several times it got completely bollixed up and I had to remove NTT completely.
This last time I have removed NTT completely several times and re-added it. But I cannot even open it now to restore it. I get the : 
![ntt](https://user-images.githubusercontent.com/31712900/34897734-9fe81f32-f7bd-11e7-957b-b4c5ba7b641a.jpg)

Is my next option to have to uninstall Firefox?
Like I said, total disaster.
**But dammit, I am determined to have the best looking New Tab Tools page ever.**

**Comment by artworksmetal** [comment]
First of all, I apologise. I'm sure I caused the problem myself by renaming image files and adding the png extension, to the displeasure of the Firefox gods. So I do not mean to disparage when I use the term buggy.
Second, I solved the issue with little discomfort by doing a 'Refresh' of FF. I had forgotten about that option.
Third, I am trying a new strategy of continuing to rename image files in order to easily assign them to tiles, but will do a Backup / restore before moving them, to hopefully 
![ntt](https://user-images.githubusercontent.com/31712900/34906759-814a3cde-f841-11e7-8962-20eadb1d844a.jpg)
appease the gods.
